cmd/bosun/web/web.go

```diff
@@ -358,12 +360,58 @@ func GetMetadata(t miniprofiler.Timer, w http.ResponseWriter, r *http.Request) (
 	return schedule.GetMetadata(r.FormValue("metric"), tags)
 }
 
+type MetricMetaTagKeys struct {
+	*database.MetricMetadata
+	TagKeys []string
+}
+
 func MetadataMetrics(t miniprofiler.Timer, w http.ResponseWriter, r *http.Request) (interface{}, error) {
```

Comment:
It Matches the underlying schedule name:

677f012e cmd/bosun/sched/sched.go (Craig Peterson 2015-09-25 14:05:28 -0600 161) func (s *Schedule) MetadataMetrics(metric string) (*database.MetricMetadata, error) {

;-)